### PR TITLE
Update ADTQueryOption documentation WRT QueryCharge

### PR DESCRIPTION
**Pull request by johngallardo**
Link QueryCharge and our Query Units documentation.

### CODE REVIEW

File: azure-monitor-ref/tables/adtqueryoperation.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ ms.date: 3/9/2023
 | OperationName | string | The type of action being performed during the event |
 | OperationVersion | string | The API Version utilized during the event |
 | ParentId | string | ParentId as part of W3C's Trace Context. A request without a parent id is the root of the trace |
-| QueryCharge | real | The QueryCharge for this event in the trace. |
+| QueryCharge | real | The charged [Query Units](https://learn.microsoft.com/en-us/azure/digital-twins/concepts-query-units) for this event in the trace |
```
Comment (johngallardo) [inline azure-monitor-ref/tables/adtqueryoperation.md]: Probably there is a better way to link this that doesn't include the en-us path fragment?

File: azure-monitor-ref/tables/adtqueryoperation.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ ms.date: 3/9/2023
 | OperationName | string | The type of action being performed during the event |
 | OperationVersion | string | The API Version utilized during the event |
 | ParentId | string | ParentId as part of W3C's Trace Context. A request without a parent id is the root of the trace |
-| QueryCharge | real | The QueryCharge for this event in the trace. |
+| QueryCharge | real | The charged [Query Units](https://learn.microsoft.com/azure/digital-twins/concepts-query-units) for this event in the trace |
```
Comment (Jak-MS) [inline azure-monitor-ref/tables/adtqueryoperation.md:43]:
```suggestion
| QueryCharge | real | The charged [Query Units](/azure/digital-twins/concepts-query-units) for this event in the trace |

```

File: azure-monitor-ref/tables/adtqueryoperation.md [hunk 3]
```diff
@@ -40,7 +40,8 @@ ms.date: 3/9/2023
 | OperationName | string | The type of action being performed during the event |
 | OperationVersion | string | The API Version utilized during the event |
 | ParentId | string | ParentId as part of W3C's Trace Context. A request without a parent id is the root of the trace |
-| QueryCharge | real | The QueryCharge for this event in the trace. |
+| QueryCharge | real | The charged [Query Units](/azure/digital-twins/concepts-query-units) for this event in the trace |
+
```
Comment (Jak-MS) [inline azure-monitor-ref/tables/adtqueryoperation.md:44]:
```suggestion
| QueryCharge | real | The charged [Query Units](/azure/digital-twins/concepts-query-units) for this event in the trace |
```

**Comment by Jak-MS** [comment]
@rboucher - accidentally merged this before your review.  let me know if you need me to revert the changes.

**Pull request merged**